projects/components/src/overlay/sheet/sheet.ts

```diff
@@ -8,5 +8,6 @@ export const enum SheetSize {
   Small = 'small',
   Medium = 'medium',
   Large = 'large',
-  ExtraLarge = 'extra-large'
+  ExtraLarge = 'extra-large',
+  Responsive = 'responsive'
```

Comment:
While responsive, this seems like it should be called halfwidth or something. Responsive itself gives no indication of size like the others do.

Reply:
Yeah. I knew you would ask this. I see responsive as a separate property than sheet size. You can have small size and have it be responsive. I discussed this with Michael as well, and there is just one use case at the moment so deferred this change for later. Like right now, we are only using one size which is Large everywhere.

Reply:
Infact talking to him today, we decided to keep it at 60%. 

how about we expose a new property as mentioned above? And how about we do it separately at a later stage ? If you agree, would you still want to rename 'Responsive' to idk `ResponsiveAt60Percent` ?

Reply:
Either something like ResponsiveMedium (mirroring the others) or making responsive a separate config like you said.

Reply:
We would need to do something like based on some breakpoints. Below one is for Extra large Sheet. So this needs more inputs from Michael and I would take it up probably next week, if its okay.
```
window size: 480, sheet size: 320… = 66%
window size: 960, sheet size: 600… = 60%
window size: 1440, sheet size: 840 = ~60%
window size: 1920, sheet size: 1280 = ~66%
```

Reply:
And now mostly responsive will be in-built and will be available to all sizes ootb.

Reply:
Renamed this to `ResponsiveExtraLarge` to match the size.